memcached.c

```diff
@@ -631,6 +635,7 @@ conn *conn_new(const int sfd, enum conn_states init_state,
     c->transport = transport;
     c->protocol = settings.binding_protocol;
 
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
with other changes below socketpath will always be 0, so this ifndef/else/endif should be removable as well.